Tests/Schema.NET.Test/StringExtensions.cs

```diff
@@ -0,0 +1,19 @@
+#if NET472
+namespace Schema.NET.Test
+{
+    using System;
+
+#pragma warning disable IDE0060 // Remove unused parameter
+#pragma warning disable CA1801 // Remove unused parameter
+
+    public static class StringExtensions
+    {
+        public static string Replace(this string target, string oldValue, string newValue, StringComparison stringComparison) => target.Replace(oldValue, newValue);
```

Comment:
We could copy the code from corefx or at least throw when `StringComparison.Ordinal` is not used for both of these methods.

Reply:
The warnings would then disappear.

Reply:
This was just to make it happy with .NET Framework as it doesn't have those methods - even where we are usingthe `StringComparison`, it isn't needed, its really just to appease the linting. If we were to throw on not having `StringComparison`, the methods that rely on it would fail for .NET Framework.